include/particle.h

```diff
@@ -330,6 +346,8 @@ class Particle : public ParticleBase<Tdim> {
   std::unique_ptr<spdlog::logger> console_;
   //! Map of vector properties
   std::map<std::string, std::function<Eigen::VectorXd()>> properties_;
+  //! Container of particle neighbour ids
+  std::vector<mpm::Index> neighbours_;
```

Comment:
Don't you think it's good to have this in `particle base`? what do you think @kks32 ?

Reply:
I am fine with it to be moved to base if necessary.

Reply:
Sure

Reply:
Done!